### DISCUSSION TITLE
Github Repository Size in API

### DISCUSSION THREAD

**Discussion by inodb** [community/community#23585] [New to GitHub]
<p>Hi all,</p>
<p>I have recently tried to make a repository smaller using BFG repo cleaner. Been working on my own fork here:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/inodb/cbioportal-frontend/">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/inodb/cbioportal-frontend/" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/1334004/181078050-92e31f40-7141-49f5-8bea-2b3326a57405.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d04747ceab8db31780948decdb0dd021bf06763_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d04747ceab8db31780948decdb0dd021bf06763_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/d/7d04747ceab8db31780948decdb0dd021bf06763.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d04747ceab8db31780948decdb0dd021bf06763_2_10x10.png"></div>

<h3><a href="https://github.com/inodb/cbioportal-frontend/" target="_blank" rel="noopener nofollow ugc">GitHub - inodb/cbioportal-frontend</a></h3>

  <p>Contribute to inodb/cbioportal-frontend development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Seems like it worked in that the repo now is &lt;70 MB when cloning down from about 1.1 GB or so:</p>
<pre><code class="lang-auto">Receiving objects: 100% (86358/86358), 66.88 MiB | 8.45 MiB/s, done.
</code></pre>
<p>But the API still reports <code>"size": 419143</code>: <a href="https://api.github.com/repos/inodb/cbioportal-frontend" rel="noopener nofollow ugc">https://api.github.com/repos/inodb/cbioportal-frontend</a>. I saw in other places that the <code>size</code> field is not very trustworthy so maybe that’s the answer here?</p>
<p>Best wishes,<br>
Ino</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>it is also different now,</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181078051-833df150-5c87-473d-8ba4-9f7c84194ce7.png" alt="image" data-base62-sha1="463O30ExoUReyafBzLRSHOhhxhs" width="229" height="69"></p>
<p>maybe it will take time to update or it’s in your cache,</p>

**Comment by inodb** [comment]
<p>Thanks! I see. Looks like there was just some delay with updating it. I see the same now</p>

**Comment by alanhamlett** [comment]
I'm still seeing `size` not accurate in the API, even when the repo hasn't had any pushes recently. For ex, one repo that's `80M` on disk has size in API of `2340` while a smaller `19M` repo has size `6414`. So `size` in the API can't be trusted?
